utils.js

```diff
@@ -1,6 +1,15 @@
 import { sep, join, resolve } from 'path';
 import { nanoid } from 'nanoid';
 import { readdir } from 'fs/promises';
+import fs from 'fs';
+
+if (!fs.existsSync('.env')) {
+  console.error('.env file not found. Please create a .env file.');
+  process.exit(1);
+}
```

Comment:
do we ever need this? if this is false it should fail on index.js already right?